Source/WebCore/animation/TimelineRange.cpp

```diff
@@ -65,11 +65,13 @@ TimelineRangeValue SingleTimelineRange::serialize() const
 
 static const std::optional<CSSToLengthConversionData> cssToLengthConversionData(RefPtr<Element> element)
 {
+    Ref document = element->document();
+    document->updateLayoutIgnorePendingStylesheets();
```

Comment:
I think this is too deep. What's the entry point from JS that we get here by?

Reply:
This is called via setBindingsRangeStart/End via the WebAnimations JS API. Moved this up into SingleTimelineRange::parse which is directly called by those setters.